doc/classes/HTTPRequest.xml

```diff
@@ -43,7 +43,7 @@
 		public override void _Ready()
 		{
 		    // Create an HTTP request node and connect its completion signal.
-		    var httpRequest = new HTTPRequest();
+		    var httpRequest = new HttpRequest();
```

Comment:
Why this change? Are the names of the C# classes invalidly generated? It's not named this

Reply:
The name of the class in C# is `HttpRequest`, not `HTTPRequest` as it would be in GDScript

Reply:
That's broken, are you certain it is the correct class? This would be an extreme exception

Reply:
Sounds like you're confusing this with the built-in C# one, which is in `System.Net` which isn't this class

Reply:
Absolutely certain. When I go to the definition in Visual Studio, I see:
```cs
[GodotClassName("HTTPRequest")]
public class HttpRequest : Node
```

Reply:
My bad, then several other places needs renaming